docs/pages/api.rst

```diff
@@ -249,27 +255,30 @@ Examples:
 ******
 MOSMIX
 ******
-::
+MOSMIX-S - less parameters:
+.. code-block:: python
 
-    from wetterdienst.mosmix.api import MOSMIXRequest
+    from wetterdienst.dwd.forecasts import DWDMosmixData, DWDFcstPeriodType
```

Comment:
Maybe let's say `DWDForecastPeriod` here? However, I don't know about other non-MOSMIX forecasts yet.

If your intention was to generalize this, then go ahead. Otherwise, maybe just say `DWDMosmixPeriod` in order to align it more specifically with the MOSMIX subsystem again.